src/bitmessagekivy/mpybit.py

```diff
@@ -206,6 +206,80 @@ def load_screen_callback(self, instance, dt=0):
             except Exception as e:
                 self.root.ids.id_trash.children[0].children[1].active = False
 
+    @staticmethod
+    def get_current_address(current_add_label, current_addr):
+        """Getting current address"""
+        addresses = [addr for addr in config.addresses()
+                     if config.getboolean(str(addr), 'enabled')]
+        if addresses:
+            if current_add_label:
+                first_name = current_add_label
+                addr = current_addr
+            else:
+                addr = addresses[0]
+                first_name = config.get(addr, 'label')
+                if not config.getboolean(addr, 'enabled'):
+                    return ''
+            address = ' (' + addr + ')'
+            return address, first_name
+        return ''
+
+    @staticmethod
+    def get_current_label(current_add_label, current_addr):
```

Comment:
```
  def format_label(label):
    returns string
```

again doesn't access config and doesn't even access `kivy_state`. Is deterministic.